pages/home.jsx

```diff
@@ -108,12 +108,47 @@ var Home = React.createClass({
             <SpeakersPromo speakersInfo={speakersInfo} />
           </div>
         </div>
-        <div className="centered content wide">
-          <h1>Missed MozFest this year?</h1>
-          <p>Sign up for MozFest updates and be first in line to get your tickets for 2017.</p>
-          <a href="https://ti.to/Mozilla/mozfest-2017" className="btn btn-arrow"><span>Sign Up</span></a>
+        <div className="news content wide">
+          <h1 className="text-center">Latest News</h1>
+          <div className="row mt-5">
+            <div className="col-12 col-md-6 text-left">
+              <div className="news-widget mb-3">
+                <img src="assets/images/simone.png" alt="Simone Browne speaking on stage at MozFest" className="mb-2" style={{ "maxWidth": "100%" }} />
+                <div className="source medium-logo text-muted">Mozilla Festival</div>
+                <a href="https://medium.com/mozilla-festival/surveillance-and-race-online" className="title">Surveillance and Race Online</a>
```

Comment:
This url is `404`. Let's change it to https://medium.com/mozilla-festival/surveillance-and-race-online-c03dda350fb9 (ugly URL, I guest c03dda350fb9 is the revision number or something? But yea this is the post we want to link to)